tests/robot/tests/3_edi_one_model/3.0_check_edi_one_model.robot

```diff
@@ -21,48 +21,6 @@ Check EDI availability in all enclaves
     \                       Should Be Equal As Integers   ${edi_state.rc}     0
     [Teardown]              NONE
 
-Get token from EDI with valid parameters
```

Comment:
@skayerok why did you delete tests for RESTful API? legionctl is not gonna be a single client for this API.